src/main/java/com/alex/poseidon/models/BidListModel.java

```diff
@@ -27,34 +31,46 @@
     private double bid;
     @Column(name = "ask")
     private double ask;
+    @Size(max=125)
     @Column(name = "benchmark")
     private String benchmark;
     @Column(name = "bid_list_date")
     private Timestamp bidListDate;
```

Comment:
What happens in the bid list is before now?

Reply:
Yes and no:
Yes: I forgot the constraints and logger in Bid list in story_1
No: I didn't pull the right version before, therefore my model classes changed (the name of model classes changed and references to it)

Reply:
Hmmmm....that doesn't solve my question, should this work if the bid_list_date has a date before now?

Reply:
I had misunderstood your question due to line 34 having a green focus. Sorry.
I have added constraint "@FutureOrPresent" for 3 fields which I judge necessary to have a check on the date (except if revisionDate is required to be filled automatically, not by the client).